tensorflow/python/ops/hidden_ops.txt

```diff
@@ -387,3 +387,6 @@ Fact
 # word2vec deprecated ops
 NegTrain
 Skipgram
+
+# dataset_ops
+SlideDataset
```

Comment:
I think you can revert this change... none of the other Dataset ops are in here.